### PR TITLE
Fix: hoppity stuff

### PR DESCRIPTION
**Pull request by CalMWolfs**
## What
 Fixed a few chocolate factory things due to hypixel update.

## Changelog Fixes
+ Fixed some Chocolate Factory issues caused by a Hypixel update. - CalMWolfs

### CODE REVIEW

**Comment by CalMWolfs** [comment]
This can be merged now as it wont break anything anymore

File: src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt [hunk 1]
```diff
@@ -52,7 +52,7 @@ object ChocolateFactoryDataLoader {
     )
     private val leaderboardPlacePattern by ChocolateFactoryAPI.patternGroup.pattern(
         "leaderboard.place",
-        "§7You are §8#§b(?<position>[\\d,]+)"
+        "(?:§.)+You are §8#§b(?<position>[\\d,]+)(?: §7in all-time Chocolate\\.)?"
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt:60]: please add a regex test

**Review by hannibal002 — CHANGES_REQUESTED** [review]
works fine ingame

File: src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt (hunk 1 above)
Comment (CalMWolfs) [reply src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryDataLoader.kt:60]: done

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**